extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/topage/ToPageWithDictionary.java

```diff
@@ -115,6 +126,13 @@ public Object nullValue() {
                 return NULL_INT;
             }
 
+            @Override
+            @NotNull
+            public PageMaterializerFactory getPageMaterializerFactory() {
+                // TODO Check if this is right
```

Comment:
TODO Discuss with Ryan

Reply:
Return a `NullFactory` that throws `UnsupportedOperationException` on use. Test that that works. ;-)